controllers/contactControllers.js

```diff
@@ -0,0 +1,90 @@
+const contactsFolder = require("../models/contactsModel");
+const {
+  addSchema,
+  updateSchema,
+} = require("../utils/validation/contactValidationSchemas");
+
+const getAll = async (req, res) => {
+  const result = await contactsFolder.find();
+  res.json(result);
+};
+
+const getByID = async (req, res) => {
+  try {
+    const { contactId } = req.params;
+    const result = await contactsFolder.findById(contactId);
```

Comment:
```suggestion
    const contact= await contactsFolder.findById(contactId);
```
давайте зміннім більш контретні назви